mathics/core/load_builtin.py

```diff
@@ -16,6 +17,11 @@
 from mathics.core.pattern import pattern_objects
 from mathics.core.symbols import Symbol
 from mathics.eval.makeboxes import builtins_precedence
+from mathics.settings import ENABLE_FILES_MODULE
+
+# List of Mathics3 Builtin modules.
+# This is initialized via below import_builtins modules
+modules = []
```

Comment:
modules is a too vague a name. Probably mathics3_builtin_modules is better.

Reply:
agree